keyboards/ortho5by12/info.json

```diff
@@ -0,0 +1,137 @@
+{
+  "keyboard_name": "ortho5by12",
+  "keyboard_folder": "ortho5by12",
+  "url": "https://github.com/itsnoteasy/misc/blob/master/ortho5by12.zip",
+  "maintainer": "gipetto",
+  "width": 12,
+  "height": 5,
+  "layouts": {
+    "LAYOUT_ortho_1x2uC": {
```

Comment:
```suggestion
    "LAYOUT_ortho_5x12_1x2uC": {
```

Reply:
no, this needs to match the name in the .json file so i can't change it without breaking a travis compile.

Reply:
[This suggestion](https://github.com/qmk/qmk_firmware/pull/9141#discussion_r428934451) changes the name used by the code, and the one here updates the JSON to match.

Not every commit needs to pass Travis CI's checks; we mainly care about the last one (some commits will cause the CI check to fail without other changes also being applied).

You can change to the [Files Changed tab](https://github.com/qmk/qmk_firmware/pull/9141/files) and batch commit suggestions, which will roll multiple suggestions into one commit:

![image](https://user-images.githubusercontent.com/18669334/82616561-70475c00-9b82-11ea-8762-21ae2d86d02d.png)

Then you'll get a button in the bar across the top of the page that will say "Commit suggestions":

![image](https://user-images.githubusercontent.com/18669334/82616651-ae448000-9b82-11ea-989e-e9aff03096ab.png)